.tekton/cert-manager-istio-csr-1-15-push.yaml

```diff
@@ -108,7 +119,7 @@ spec:
       name: build-args-file
       type: string
     - default:
-      - linux/x86_64
+      - $(params.build-platforms)
```

Comment:
This is missed in pull-request yaml, please update it.

Same is missed in `https://github.com/openshift/cert-manager-operator-release/blob/release-1.15/.tekton/cert-manager-operator-1-15-pull-request.yaml` . If you don't mind, could you please update it as well, but with a different commit.

Reply:
Thanks @bharath-b-rh For pointing this out Added the suggestion in a different commit!